src/main/java/seedu/duke/Duke.java

```diff
@@ -65,6 +65,8 @@ public void run() {
                 System.out.println(m.getMessage());
             }
         } while (isNotBye(command));
+
+        ui.close();
```

Comment:
Why must we close the scanner? (Just a question to learn)

Reply:
https://www.quora.com/Why-do-we-need-to-close-the-scanner-class-in-java?share=1

Btw, I didn't know why need to close until now 😃